DEV-309: Remove the notion of 'Teams' from status indicator frontend and backend (#444)
highfidelity/hifi-content: usefulUtilities/statusIndicator/README.md

# Status Indicator
Let other people know your status!

> This feature consists of a High Fidelity Interfaceclient script and a backend Node.js Web application.

Reviewer: ```suggestion
This feature consists of a High Fidelity Interface client script and a backend Node.js Web application.
```

> The client script creates a clickable button in upper right hand corner of your screen to set your status to "available" or "busy". The button only appears in Desktop mode.

Reviewer: ```suggestion
The client script creates a clickable button in the upper right-hand corner of your screen to set your status to "available" or "busy". The button only appears in Desktop mode.
```

}
```

* During development and local testing, this REQUEST_URL was `http://localhost:3305/`

> Once `secrets.json` is set up, run `<statusIndicator root>/hifiScript/statusIndicator.js` in High Fidelity's Running scripts.

Reviewer: Does "scripts" need to be capitalized?

Author: ```suggestion
Once `secrets.json` is set up, run `<statusIndicator root>/hifiScript/statusIndicator.js` in High Fidelity's Running Scripts.
```

    "mySQLUsername": "<MySQL DB username>",
    "mySQLPassword": "<MySQL DB password>",
    "databaseName": "<the name of the database associated with the Status Indicator app>",

>     "wwwRoot": "<a URL to the remote host serving the files in `/webApp/www/`. include the trailing slash. used for testing so that the `canary` API endpoint knows where to check. example: https://example.com/statusIndicator/>"

Reviewer: ```suggestion
    "wwwRoot": "<a URL to the remote host serving the files in `/webApp/www/`. Include the trailing slash. Used for testing so that the `canary` API endpoint knows where to check. Example: https://example.com/statusIndicator/>"
```

Author: stylistically, i'm keeping all of these sentences lower-case. i don't have a super good reason for it, but i'm consistent :)
